src/node/glossary.ts

```diff
@@ -0,0 +1,29 @@
+import { SharedOptions } from '../sharedOptions'
+import { RequestHandlersList } from '../setupWorker/glossary'
+
+export interface SetupServerApi {
```

Comment:
To make the interface accessibile we should export from `node/index.ts`

Reply:
Thank you for the suggestion. Addressed.